sql/sqlwire/sql_api.proto

```diff
@@ -93,6 +93,9 @@ message SQLRequest {
 
 message SQLResponse {
   optional SQLResponseHeader header = 1 [(gogoproto.nullable) = false, (gogoproto.embed) = true];
+  // The names of the columns returned in the result set.
+  // The number of columns must equal the number of Datum in each Result.
```

Comment:
This is bordering on pedantic, but I don't see any mention here or on the other Response/Result types about how the columns in each row are ordered; I'm assuming there is a guaranteed order of columns shared by this "columns" set and the values in every `Result` message.